app/src/main/java/com/yagni/model/ReadAllOrders.java

```diff
@@ -3,6 +3,7 @@
 import java.sql.*;
 import javax.swing.*;
 import javax.swing.table.DefaultTableModel;
+
```

Comment:
delete line 6 whitespace. import com.yagni.controller.*; should be grouped with other imports